build/maven_test.go

```diff
@@ -72,3 +72,52 @@ func getExpectedMavenBuildInfo(t *testing.T, filePath string) entities.BuildInfo
 	assert.NoError(t, json.Unmarshal(data, &buildinfo))
 	return buildinfo
 }
+
+func TestExtractMavenPath(t *testing.T) {
+	service := NewBuildInfoService()
+	mavenBuild, err := service.GetOrCreateBuild("build-info-maven-test", "1")
+	assert.NoError(t, err)
+	defer func() {
+		assert.NoError(t, mavenBuild.Clean())
+	}()
```

Comment:
```suggestion
	defer assert.NoError(t, mavenBuild.Clean())
```

Reply:
I tried to change this, then the test is not passing any more

Reply:
I disagree. This is exactly the same in that case.